### DISCUSSION TITLE
I translated @zadam's wiki into chinese😁

### DISCUSSION THREAD

**Discussion by baddate** [TriliumNext/Trilium#1499] [Show and tell]
here it is:

https://github.com/baddate/trilium/wiki

**Comment by Waxwood** [comment]
Excellent work! Thank you for doing this! 
There's a huge number of Chinese language speakers who will benefit from this! 
This may help expand Trilium's user base and assist with its development.

**Comment by sepwolves** [comment]
想不到用这个软件的中国人还挺多的哈。

**Reply by Waxwood** [reply]
希望橫多人會用Trilium，能幫助zadam擴大影響和這軟件的功能。

**Comment by sepwolves** [comment]
有没有什么比较热烈的讨论区呢？这里太冷淡了，参与的人太少了。
